src/test/java/seedu/address/logic/parser/SortCommandParserTest.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.Comparator;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.SortCommand;
+import seedu.address.model.person.Person;
+
+public class SortCommandParserTest {
+    private final SortCommandParser parser = new SortCommandParser();
+
+    @Test
+    public void parse_singlePrefix_success() {
```

Comment:
1. Prefixes will work on their own
2. Prefixes with ascending
3. Prefixes with descending

Reply:
Could possibly add double prefix tests, but might slow down the testing phase.